test/User-test.js

```diff
@@ -45,74 +47,120 @@ describe("User", () => {
-    expect(userData.newUser[1].name).to.equal("Ephraim Goyette");
+    expect(user1.name).to.equal("Saige O'Kon");
   });
 
   it("should have an id", () => {
-    expect(userData.newUser[0].id).to.equal(1);
-    expect(userData.newUser[1].id).to.equal(2);
+    expect(user1.id).to.equal(1)
   });
 
   it("should have a pantry array", () => {
-    expect(userData.newUser[0].pantry).to.be.an("array");
-    expect(userData.newUser[1].pantry).to.be.an("array");
+    expect(user1.pantry).to.be.an("array");
+  
   });
 
   it("should have an object in pantry", () => {
-    expect(userData.newUser[0].pantry[0]).to.be.an("object");
-    expect(userData.newUser[1].pantry[1]).to.be.an("object");
+    expect(user1.pantry[0]).to.be.an("object");
   });
 
   it("should have ingredient numbers", () => {
-    expect(userData.newUser[0].pantry[1].ingredient).to.equal(1082047);
-    expect(userData.newUser[1].pantry[2].ingredient).to.equal(1082047);
+    expect(user1.pantry[1].ingredient).to.equal(1082047);
   });
 
   it("should have ingredient amounts", () => {
-    expect(userData.newUser[0].pantry[0].amount).to.equal(4);
-    expect(userData.newUser[1].pantry[0].amount).to.equal(3);
+    expect(user1.pantry[0].amount).to.equal(4);
+    expect(user2.pantry[0].amount).to.equal(3);
   });
 
-  it.skip("should pull a recipe from the recipe scripts", () => {
-    console.log(recipeData);
-    expect(recipeToCook()).to.be.equal(recipeData[0].id);
-  });
+  // it("should pull a recipe from the recipe scripts", () => {
+  //   console.log(recipeData);
+  //   expect(recipeToCook()).to.be.equal(recipeData[0].id);
+  // });
 
-  it.skip("should add a recipe to an array", () => {
-    expect(array.length).to.equal(0);
-    recipeToCook();
+  it("should add a recipe to an array", () => {
+    expect(user1.toCook.length).to.equal(0);
+    //need to call as a method
+    //method takes the "recipe" object  as a parameter
+    //converts to Recipe class and pushes to this.toCook array
+    user1.recipeToCook(recipe1)
+    expect(user1.toCook[0]).to.equal(recipe1)
 
-    expect(array.length).to.equal(1);
+    expect(user1.toCook.length).to.equal(1);
   });
 
-  it.skip("should contain an object in the array of recipes", () => {
-    recipeToCook();
-    expect(array[0]).to.be.an("object");
+  it("should contain an object in the array of recipes", () => {
+    user1.recipeToCook(recipe1)
+    expect(user1.toCook[0]).to.be.an("object");
   });
 
-  it.skip("should delete an object from the array of recipes", () => {
-    recipeToCook();
-    expect(array.length).to.equal(1);
+  it("should delete an object from the array of recipes", () => {
+    user1.recipeToCook(recipe1);
+    expect(user1.toCook.length).to.equal(1);
+    expect(user1.toCook[0].name).to.equal("Loaded Chocolate Chip Pudding Cookie Cups")
 
-    deleteRecipeToCook();
-    expect(array.length).to.equal(0);
+    user1.removeRecipeToCook(recipe1);
+    expect(user1.toCook.length).to.equal(0);
   });
```

Comment:
-removeRecipeToCook(recipe) takes in a recipe as argument, it then removes it from this.toCook array
-looking at it now i don't think i tested this one robustly enough
-we may even need to add another stand in recipe object to the test file, pass both of them in, and make sure its picking the specified recipe